packages/cli/Config.graphqls

```diff
@@ -0,0 +1,6 @@
+extend input GraphCommerceConfig {
+  """
+  Hygraph Management SDK Authorization Token
+  """
+  hygraphAuthToken: String
```

Comment:
Rename to `hygraphWriteAccessToken` but never configure it, but provide it with an env variable: `GC_HYGRAPH_WRITE_ACCESS_TOKEN="123"` (I'll might introduce a way to mark these as secrets so they aren't allowed to be added to `graphcommerce.config.js`)